tests/common/devices.py

```diff
@@ -324,5 +324,6 @@ def __init__(self, ansible_adhoc, hostname, user, passwd, gather_facts=False):
         evars = { 'ansible_connection':'network_cli', \
                   'ansible_network_os':'eos', \
                   'ansible_user': user, \
-                  'ansible_password': passwd }
+                  'ansible_password': passwd, \
+                  'ansible_become_method': 'enable' }
```

Comment:
This is required to execute eos_config module, but I'm not quite sure if grant privilege by default is OK.
If you not suggest add this privilege here, I can add a class method to let user grant privilege explicitly.